docs/Fleet-provisioning.md

```diff
@@ -9,23 +9,33 @@ can get valid certificates. you can follow the link
 [here](https://docs.aws.amazon.com/greengrass/v2/developerguide/fleet-provisioning-setup.html)
 to learn how to create appropriate policies and claim certificate.
 
-```
-Note:
-Currently, fleet provisioning can only be run manually.
-Hence you will need to follow few important pre-steps
-
-1. Make sure you are logged in as root
-2. Allow read access to all user for your certificates
-    chmod -R +rx /ggcredentials/
-3. Make sure you do not fill iotCredEndpoint/iotDataEndpoint under
-  `aws.greengrass.NucleusLite` you should only fill these fields
-  under `aws.greengrass.fleet_provisioning`'s config
-4. If this is your not first run, remove the socket at
-    /run/greengrass/iotcoredfleet, if it exists
-5. Fleet provisioning assumes the your GGL_SYSTEMD_SYSTEM_USER
-  and GGL_SYSTEMD_SYSTEM_GROUP to be ggcore:ggcore please change
-  appropriately if you change these values during compile time
-```
+Greengrass nucleus lite generates csr and private keys locally and then sends
+the csr to iotcore to generate a certificate. This behavior is different from
+Greengrass classic. Hence make sure your claim certificate have connect,
+publish, subscribe and receive access to `CreateCertificateFromCsr` and
+`RegisterThing` topics mentioned in
+[linked AWS docs](https://docs.aws.amazon.com/iot/latest/developerguide/fleet-provision-api.html).
+
+## Before getting started:
```

Comment:
Everything is in this section, I think after the configs are set up a new section for running the fleet provisioning process should be inserted